invenio_db/alembic/dbdbc1b19cf2_create_transaction_table.py

```diff
@@ -40,8 +41,9 @@ def upgrade():
         sa.Column('issued_at', sa.DateTime(), nullable=True),
         sa.Column('id', sa.BigInteger(), nullable=False),
         sa.Column('remote_addr', sa.String(length=50), nullable=True),
-        sa.PrimaryKeyConstraint('id')
     )
+    op.execute(CreateSequence(Sequence("transaction_id_seq")))
```

Comment:
@jirikuncar right, I added autoincrement=True and `flask db drop` works fine after the alembic commands, but there is no sequence created:
```
+-------------+-----------------------------+-------------+
| Column      | Type                        | Modifiers   |
|-------------+-----------------------------+-------------|
| issued_at   | timestamp without time zone |             |
| id          | bigint                      |  not null   |
| remote_addr | character varying(50)       |             |
+-------------+-----------------------------+-------------+
Indexes:
    "pk_transaction" PRIMARY KEY, btree (id)

Time: 0.015s
postgres@localhost:invenio> \ds
+----------+--------+--------+---------+
| Schema   | Name   | Type   | Owner   |
|----------+--------+--------+---------|
+----------+--------+--------+---------+
```
 so it differs slightly from `flask db create`.